.github/ISSUE_TEMPLATE/project-onboarding.md

```diff
@@ -0,0 +1,19 @@
+---
+name: project onboarding
+about: project onboarding
+title: "[PROJECT ONBOARDING] project"
+labels: project onboarding
+---
+
+Welcome to Argo Project Onboarding! 
+
+Before submitting the ticket please make sure to understand which [projects](../../community/ecosystem-projects.md) can be added to Argoproj organization and get familiar with the onboarding process. 
+
+Once you are ready, provide answers to the following questions and create the issue:
+
```

Comment:
Perhaps we should also require a couple of sponsors from the subproject.
Require a short description of the project.

Reply:
Good idea. Added